akka-distributed-data/src/main/scala/akka/cluster/ddata/DeltaPropagationSelector.scala

```diff
@@ -104,7 +105,13 @@ import akka.cluster.ddata.Replicator.Internal.DeltaPropagation
               val deltaGroup = cache.get(cacheKey) match {
                 case None ⇒
                   val group = deltaEntriesAfterJ.valuesIterator.reduceLeft {
-                    (d1, d2) ⇒ d1.merge(d2.asInstanceOf[d1.T])
+                    (d1, d2) ⇒
+                      d2 match {
+                        case NoDeltaPlaceholder ⇒ NoDeltaPlaceholder
```

Comment:
why `NoDeltaPlaceholder` and not `d1` ? Would the `NoDeltaPlaceholder` not cancel out the already accumulated value?